src/main/java/taskmaster/exceptions/DeadlineCommandMissingInputException.java

```diff
@@ -0,0 +1,5 @@
+package taskmaster.exceptions;
+
+public class DeadlineCommandMissingInputException extends Exception {
+
```

Comment:
consider having some custome exception handling - at least an error message to indicate what went wrong.

Reply:
Same for other exceptions below